samples/meeting-av-filters/csharp/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-meeting-av-filters-csharp",
+    "source": "officeDev",
+    "title": "Meeting av filter",
+    "shortDescription": "This sample illustrates usage of av filters for teams meetings.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meeting-av-filters/csharp",
+    "longDescription": [
+      "This sample illustrates usage of av filters for teams meetings."
+    ],
+    "creationDateTime": "2022-08-30",
+    "updateDateTime": "2022-08-30",
+    "products": [
+      "Teams"-
+    ],
+    "metadata": [
+      {
+        "key": "TEAMS-SAMPLE-SOURCE",
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": "csharp"
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": "netframework"
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "av-filter"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/main/samples/incoming-webhook/csharp/IncomingWebhook/Images/cardByWebhook.png",
```

Comment:
URL is also wrong ?

Reply:
The sample is still not published and merged with main in github so kept the previous URL